tests/by-util/test_tail.rs

```diff
@@ -4824,3 +4824,40 @@ fn test_obsolete_encoding_windows() {
         .stderr_is("tail: bad argument encoding: '-�b'\n")
         .code_is(1);
 }
+
+#[test]
+fn test_following_with_pid() {
+    use std::process::Command;
+    use std::time::Duration;
+    use std::thread::sleep;
+
+    let scene = TestScenario::new(util_name!());
+
+    let sleep_command = Command::new("sleep")
+        .arg("10")
```

Comment:
Your setup doesn't require that `sleep` ever terminates normally, so why did you choose only 10 seconds?
```suggestion
        .arg("999d")
```